src/main/kotlin/io/github/mojira/arisa/modules/commands/DeleteLinksCommand.kt

```diff
@@ -0,0 +1,25 @@
+package io.github.mojira.arisa.modules.commands
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+import io.github.mojira.arisa.domain.Issue
+import io.github.mojira.arisa.modules.ModuleError
+import io.github.mojira.arisa.modules.ModuleResponse
+import io.github.mojira.arisa.modules.assertTrue
+import io.github.mojira.arisa.modules.assertFalse
+import io.github.mojira.arisa.modules.concatLinkName
+import io.github.mojira.arisa.modules.convertLinks
+import io.github.mojira.arisa.modules.deleteLinks
+
+class DeleteLinksCommand : Command {
+    override fun invoke(issue: Issue, vararg arguments: String): Either<ModuleError, ModuleResponse> = Either.fx {
+        assertTrue(arguments.size > 2).bind()
+        val tmpArray = listOf(*arguments).subList(1, arguments.size).toTypedArray()
+        var args = concatLinkName(*tmpArray)
+        val type = args[0]
+        assertFalse(type == "").bind()
+        args = listOf(*args).subList(1, args.size).toTypedArray()
```

Comment:
When you have to use var, its generally a bad idea 👀, Put this in another function 😅

Reply:
it happened 'cos I held everything in Array, which doesn't support kotlin list functions
I guess it would be better idea to pass lists as arguments then?

Otherwise I'm not sure how you would put that in a separate function <.<

Reply:
If you are operating on it its probably better to pass as a list

Reply:
alright, gonna fix it then

Reply:
fixed by https://github.com/mojira/arisa-kt/pull/428/commits/6562b9f3985be0b05b318db5263c17b429cacee5